tests/10-http-signatures.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+ /* globals describe, before, after, it, should, beforeEach, afterEach */
+ /* jshint node: true */
+'use strict';
+
+var _ = require('lodash');
+var async = require('async');
+var bedrock = require('bedrock');
+var brKey = require('bedrock-key');
+var config = bedrock.config;
+var helpers = require('./helpers');
+var brIdentity = require('bedrock-identity');
+var database = require('bedrock-mongodb');
+var mockData = require('./mock.data');
+var store = require('bedrock-credentials-mongodb').provider;
+var uuid = require('node-uuid').v4;
+var url = require('url');
+var util = bedrock.util;
+var request = require('request');
+request = request.defaults({json: true, strictSSL: false});
+
+var urlObj = {
+  protocol: 'https',
+  host: config.server.host,
+  pathname: '/tests/bedrock-passport/http-signature-test'
+};
+
+describe('bedrock-passport', function() {
+  before('Prepare the database', function(done) {
+    helpers.prepareDatabase(mockData, done);
```

Comment:
For this specific test -- no, I don't think it's needed, but any other tests are going to want to be able to work with identities that are in the DB.